npm failure

## Npm fails because of permission.
```
―― 20:47:24 - Node Package Manager ―――――――――――――――――――――――――――――――――――――――――――――
npm ERR! code EACCES
npm ERR! syscall rename
npm ERR! path /usr/lib/node_modules/is-stream
npm ERR! dest /usr/lib/node_modules/.is-stream-ibQnO7oZ
npm ERR! errno -13
npm ERR! Error: EACCES: permission denied, rename '/usr/lib/node_modules/is-stream' -> '/usr/lib/node_modules/.is-stream-ibQnO7oZ'
npm ERR!  [Error: EACCES: permission denied, rename '/usr/lib/node_modules/is-stream' -> '/usr/lib/node_modules/.is-stream-ibQnO7oZ'] {
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'rename',
npm ERR!   path: '/usr/lib/node_modules/is-stream',
npm ERR!   dest: '/usr/lib/node_modules/.is-stream-ibQnO7oZ'
npm ERR! }
npm ERR! 
npm ERR! The operation was rejected by your operating system.
npm ERR! It is likely you do not have the permissions to access this file as the current user
npm ERR! 
npm ERR! If you believe this might be a permissions issue, please double-check the
npm ERR! permissions of the file and its containing directories, or try running
npm ERR! the command again as root/Administrator.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/said/.npm/_logs/2022-11-10T17_47_24_556Z-debug-0.log
```

## Specifications
OS: Pop!_OS 22.04 LTS x86_64
Topgrade version 10.1.2 unknown linux GNU, downloaded from releases

## If I run `sudo npm update`
```
❯ sudo npm update

up to date, audited 36 packages in 4s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

--- a/src/steps/node.rs
+++ b/src/steps/node.rs
@@ -91,7 +91,11 @@ impl NPM {
     fn upgrade(&self, run_type: RunType, use_sudo: bool) -> Result<()> {
         let args = ["update", self.global_location_arg()];
         if use_sudo {
-            run_type.execute("sudo").args(args).status_checked()?;
+            run_type
+                .execute("sudo")
+                .arg(&self.command)
+                .args(args)
+                .status_checked()?;
         } else {
             run_type.execute(&self.command).args(args).status_checked()?;
         }

